.github/workflows/containerd.yml

```diff
@@ -10,7 +10,24 @@ jobs:
       matrix:
         os: [ubuntu-18.04, windows-2019]
         version: [master, v1.4.0-beta.2]
-    name: ${{matrix.version}} / ${{ matrix.os }}
+        runtime: [io.containerd.runtime.v1.linux, io.containerd.runc.v1, io.containerd.runc.v2, containerd-shim-runhcs-v1]
+        runc: [runc, crun]
+        exclude:
+          - runtime: io.containerd.runc.v1
```

Comment:
feels like a text table would be nice here :-)

Reply:
Added